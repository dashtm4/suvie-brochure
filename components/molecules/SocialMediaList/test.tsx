import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SocialMediaList } from '.'

describe('<SocialMediaList />', () => {
  it('should has correct child', () => {
    const socialMediaList = [
      {
        title: 'Facebook',
        url: 'https://www.facebook.com/SuvieFood/',
        icon: '/images/facebook-icon.svg'
      },
      {
        title: 'Twitter',
        url: 'https://twitter.com/suviefood',
        icon: '/images/twitter-icon.svg'
      }
    ]

    const wrapper = shallow(<SocialMediaList list={socialMediaList} />)

    expect(wrapper.find('ul.flex.w-full li.mr-2 SocialMediaIcon')).toHaveLength(
      2
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
