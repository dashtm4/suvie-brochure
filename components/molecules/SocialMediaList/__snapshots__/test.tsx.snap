// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialMediaList /> should has correct child 1`] = `
<ul
  className="flex w-full"
>
  <li
    className="mr-2"
    key="Facebook"
    role="presentation"
  >
    <SocialMediaIcon
      key="Facebook"
      socialMedia={
        Object {
          "icon": "/images/facebook-icon.svg",
          "title": "Facebook",
          "url": "https://www.facebook.com/SuvieFood/",
        }
      }
    />
  </li>
  <li
    className="mr-2"
    key="Twitter"
    role="presentation"
  >
    <SocialMediaIcon
      key="Twitter"
      socialMedia={
        Object {
          "icon": "/images/twitter-icon.svg",
          "title": "Twitter",
          "url": "https://twitter.com/suviefood",
        }
      }
    />
  </li>
</ul>
`;
